server/http4s-server/src/main/scala/sttp/tapir/server/http4s/Http4sServerInterpreter.scala

```diff
@@ -23,102 +23,60 @@ import sttp.tapir.server.interpreter.{BodyListener, ServerInterpreter}
 
 import scala.reflect.ClassTag
 
-trait Http4sServerInterpreter {
-  def toHttp[I, E, O, F[_], G[_]](
-      e: Endpoint[I, E, O, Fs2Streams[F] with WebSockets]
-  )(fToG: F ~> G)(gToF: G ~> F)(logic: I => G[Either[E, O]])(implicit
-      serverOptions: Http4sServerOptions[F, G],
-      gs: Sync[G],
-      fs: Concurrent[F],
-      fcs: ContextShift[F],
-      timer: Timer[F]
-  ): Http[OptionT[G, *], F] = toHttp(e.serverLogic(logic))(fToG)(gToF)
-
-  def toHttpRecoverErrors[I, E, O, F[_], G[_]](
-      e: Endpoint[I, E, O, Fs2Streams[F] with WebSockets]
-  )(fToG: F ~> G)(gToF: G ~> F)(logic: I => G[O])(implicit
-      serverOptions: Http4sServerOptions[F, G],
-      gs: Sync[G],
-      fs: Concurrent[F],
-      fcs: ContextShift[F],
-      eIsThrowable: E <:< Throwable,
-      eClassTag: ClassTag[E],
-      timer: Timer[F]
-  ): Http[OptionT[G, *], F] = toHttp(e.serverLogicRecoverErrors(logic))(fToG)(gToF)
-
-  def toRoutes[I, E, O, F[_]](e: Endpoint[I, E, O, Fs2Streams[F] with WebSockets])(
-      logic: I => F[Either[E, O]]
-  )(implicit serverOptions: Http4sServerOptions[F, F], fs: Concurrent[F], fcs: ContextShift[F], timer: Timer[F]): HttpRoutes[F] = toRoutes(
-    e.serverLogic(logic)
-  )
-
-  def toRouteRecoverErrors[I, E, O, F[_]](e: Endpoint[I, E, O, Fs2Streams[F] with WebSockets])(logic: I => F[O])(implicit
-      serverOptions: Http4sServerOptions[F, F],
-      fs: Concurrent[F],
-      fcs: ContextShift[F],
-      eIsThrowable: E <:< Throwable,
-      eClassTag: ClassTag[E],
-      timer: Timer[F]
-  ): HttpRoutes[F] = toRoutes(e.serverLogicRecoverErrors(logic))
+trait Http4sServerInterpreter[F[_], G[_]] {
```

Comment:
I think the `toRoutes` variant is the one that is predominantly used. So maybe we could rename this one to `Http4sServerToHttpInterpreter`, and the routes one would keep the simpler `Http4sServerInterpreter` name?